packages/icon/src/css/index.js

```diff
@@ -10,7 +10,14 @@ import * as vars from '../vars/index.js'
 
 export default {
   [`.psds-icon`]: {
-    display: 'inline-block'
+    display: 'inline-flex',
```

Comment:
Not likely, but since Icon can take a child, is this layout change potentially breaking for any of those custom child cases?  (the ones that we don't have in our storybook to check)

Reply:
Not that I can think of given Icon takes default size of medium